scanmem.1

```diff
@@ -443,6 +448,19 @@ ugtrain are required to periodically refill variables is such memory regions.
 
 .SH BUGS
 
+.B scanmem
+usually requires root privileges for
+.BR ptrace (2)
+because security modules control ptrace() capabilities. On x86 and x86_64 there is usually
+the
+.B Yama
+security module providing the file
+.IR /proc/sys/kernel/yama/ptrace_scope "."
+It is available since Linux 3.4. If this file contains "1", then only parents may ptrace()
+their children without root privileges. This means that scanmem would have to run the
+game. This is not possible as this would require major design changes. So we run scanmem
```

Comment:
I rather had the feeling that there is too much bold for `scanmem` in the man page and I thought it is enough to do it at least once per paragraph. Should I make it always bold?